AsyncDisplayKit/ASControlNode.mm

```diff
@@ -82,9 +82,21 @@ - (id)init
 
   // As we have no targets yet, we start off with user interaction off. When a target is added, it'll get turned back on.
   self.userInteractionEnabled = NO;
+  
   return self;
 }
 
+#if TARGET_OS_TV
+- (void)didLoad
+{
+  //On tvOS all control views, such as buttons, interact with the focus system even if they don't have a target set on them. Here we add our own internal tap gesture to handle this behaviour.
```

Comment:
maybe just say all controls, rather than control views.
Create a second line for the comment rather than long line, and always space after //